### PR TITLE
Rtf 1262 production data source strategies

### PR DESCRIPTION
**Pull request by maria-kizhaeva**
Complete the following tasks before merging in your PR:
- [x] If this is a user facing change, update [version.txt](https://github.com/relativitydev/relativity.testing.framework.api/blob/master/version.txt) and [CHANGELOG.md](https://github.com/relativitydev/relativity.testing.framework.api/blob/master/CHANGELOG.md)

### CODE REVIEW

File: source/Relativity.Testing.Framework.Api/Strategies/Productions/ProductionsDataSourceCreateStrategyPreOsier.cs [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ﻿using System;
 using Relativity.Testing.Framework.Api.Services;
 using Relativity.Testing.Framework.Models;
-
+using Relativity.Testing.Framework.Versioning;
+
 namespace Relativity.Testing.Framework.Api.Strategies
 {
+	[VersionRange("<12.1")]
 	internal class ProductionsDataSourceCreateStrategy : ICreateWorkspaceEntityStrategy<ProductionDataSource>
```
Comment (paulletang) [inline source/Relativity.Testing.Framework.Api/Strategies/Productions/ProductionsDataSourceCreateStrategyPreOsier.cs:9]: The class name should change to match the file.

File: source/Relativity.Testing.Framework.Api/Services/ProductionDataSourceService.cs [hunk 2]
```diff
@@ -40,5 +43,8 @@ public void Update(int workspaceId, int productionId, ProductionDataSource entit
 
 		public ProductionDataSource Get(int workspaceId, int entityId)
 			=> _getWorkspaceEntityByIdStrategy.Get(workspaceId, entityId);
+
+		public ProductionDataSourceDefaultValues GetDefaultFieldValues(int workspaceId)
+		=> _getDefaultFieldValuesStrategy.Get(workspaceId);
```
Comment (paulletang) [inline source/Relativity.Testing.Framework.Api/Services/ProductionDataSourceService.cs:48]: It looks like the indentation is off on this line.

File: source/Relativity.Testing.Framework.Api.FunctionalTests/Strategies/Productions/ProductionsDataSourceGetDefaultFieldValuesStrategyFixture.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+		public void Get_ShouldNotThrown()
+		{
+			Assert.DoesNotThrow(() => Sut.Get(DefaultWorkspace.ArtifactID));
+		}
+
+		[Test]
+		public void Get_ResponseNotEmpty()
+		{
+			var result = Sut.Get(DefaultWorkspace.ArtifactID);
+			Assert.AreNotEqual(0, result.BurnRedactions.ArtifactID);
```
Comment (briantoms) [inline source/Relativity.Testing.Framework.Api.FunctionalTests/Strategies/Productions/ProductionsDataSourceGetDefaultFieldValuesStrategyFixture.cs:19]: If this is a boolean, why does it have an `ArtifactID`?

File: source/Relativity.Testing.Framework.Api.FunctionalTests/Strategies/Productions/ProductionsDataSourceGetDefaultFieldValuesStrategyFixture.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
+﻿using NUnit.Framework;
+using Relativity.Testing.Framework.Api.Strategies;
+
+namespace Relativity.Testing.Framework.Api.FunctionalTests.Strategies
+{
+	[TestOf(typeof(IProductionsDataSourceGetDefaultFieldValuesStrategy))]
+	internal class ProductionsDataSourceGetDefaultFieldValuesStrategyFixture : ApiServiceTestFixture<IProductionsDataSourceGetDefaultFieldValuesStrategy>
+	{
+		[Test]
+		public void Get_ShouldNotThrown()
```
Comment (paulletang) [inline source/Relativity.Testing.Framework.Api.FunctionalTests/Strategies/Productions/ProductionsDataSourceGetDefaultFieldValuesStrategyFixture.cs:10]: Extra `n` at the end.

File: source/Relativity.Testing.Framework.Api.FunctionalTests/Strategies/Productions/ProductionPlaceholderGetDefaultFieldValuesStrategyFixture.cs [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ﻿using NUnit.Framework;
 using Relativity.Testing.Framework.Api.Strategies;
-
+using Relativity.Testing.Framework.Models;
```
Comment (paulletang) [inline source/Relativity.Testing.Framework.Api.FunctionalTests/Strategies/Productions/ProductionPlaceholderGetDefaultFieldValuesStrategyFixture.cs:3]: Is this in use now?
It doesn't look like anything else changed in this file.

File: source/Relativity.Testing.Framework.Api.FunctionalTests/Strategies/Productions/ProductionsDataSourceGetDefaultFieldValuesStrategyFixture.cs (hunk 3 above)
Comment (maria-kizhaeva) [reply source/Relativity.Testing.Framework.Api.FunctionalTests/Strategies/Productions/ProductionsDataSourceGetDefaultFieldValuesStrategyFixture.cs:19]: It's some sort of a predefined object with ArtifactId that has some boolean default value.

Comment (briantoms) [reply source/Relativity.Testing.Framework.Api.FunctionalTests/Strategies/Productions/ProductionsDataSourceGetDefaultFieldValuesStrategyFixture.cs:19]: Is this the ArtifactID of the _field_? According to the API docs, getting the default field values should return, for each field, the field GUID, the field ArtifactID, and the default value (could be bool, Choice, etc.).

The model's definitely not intuitive here, but I don't know how I would change it to be clearer.

**Review by klaudiaknafel — APPROVED** [review]

**Review by jonatanjakoniuk — APPROVED** [review]

**Pull request merged**